C-Plus-Plus/other/Kth largest element.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+Q Kth largest element 
+
+Given an array a[] and a number K where K is smaller than size of array, 
+the task is to find the Kth largest element in the given array.
+It is given that all array elements are distinct. */
+
+
+#include <bits/stdc++.h>
+#include <iostream>
```

Comment:
You don't need to add ```#include <iostream>``` if you have already added ```#include <bits/stdc++.h>```. Make the change in both the programs.